apps/aecontract/src/aect_dispatch.erl

```diff
@@ -133,41 +133,50 @@ call_common(#{ caller     := CallerPubKey
         InitState ->
             %% Update gas_used depending on exit type.
             try aevm_eeevm:eval(InitState) of
-                {ok, #{gas := GasLeft, out := Out, chain_state := ChainState}} ->
+                {ok, ResultState} ->
+                    GasLeft = aevm_eeevm_state:gas(ResultState),
+                    Out = aevm_eeevm_state:out(ResultState),
+                    ChainState = aevm_eeevm_state:chain_state(ResultState),
+                    Log = aevm_eeevm_state:logs(ResultState),
                     {
-                        create_call(Gas - GasLeft, ok, Out, Call),
-                        aec_vm_chain:get_trees(ChainState)
+                      create_call(Gas - GasLeft, ok, Out, Log, Call),
+                      aec_vm_chain:get_trees(ChainState)
                     };
-                {revert, #{gas := GasLeft, out := Out, chain_state := ChainState}} ->
+                {revert, ResultState} ->
+                    GasLeft = aevm_eeevm_state:gas(ResultState),
+                    Out = aevm_eeevm_state:out(ResultState),
+                    ChainState = aevm_eeevm_state:chain_state(ResultState),
+                    Log = aevm_eeevm_state:logs(ResultState),
                     {
-                        create_call(Gas - GasLeft, revert, Out, Call),
+                        create_call(Gas - GasLeft, revert, Out, Log, Call),
```

Comment:
I need to check if there is a test for this (I expect no). I was expecting `[]` here.

Reply:
I changed this to `[]` and ran tests. No failure. So I think that checking this shall be tracked.

Reply:
Tracked as https://www.pivotaltracker.com/story/show/160721291